sample/DocumentDB-CSharp/run.csx

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+public static void Run(string input, out object newItem)
+{
+    newItem = new 
+    {
+        text = "Hello from C#! " + input
+    }; 
+}
```

Comment:
I agree :-)

---
In reply to: [56433393](https://github.com/Azure/azure-webjobs-sdk-script/pull/112#discussion_r56433393) [](ancestors = 56433393)